Multisig/UI/Settings/OwnerKeyManagement/OnboardingLedgerKeyViewController.swift

```diff
@@ -32,6 +32,29 @@ class OnboardingLedgerKeyViewController: AddKeyOnboardingViewController {
 
     override func didTapNextButton(_ sender: Any) {
         let vc = SelectLedgerDeviceViewController()
+        vc.delegate = self
         show(vc, sender: self)
     }
 }
+
+extension OnboardingLedgerKeyViewController: SelectLedgerDeviceDelegate {
+    func selectLedgerDeviceViewController(_ controller: SelectLedgerDeviceViewController,
+                                          didSelectDevice deviceId: UUID,
+                                          bluetoothController: BluetoothController) {
+
+        let ledgerController = LedgerController(bluetoothController: bluetoothController)
+        ledgerController.getAddress(deviceId: deviceId, at: 10) { [weak controller] ledgerInfoOrNil in
+            guard let ledgerInfo = ledgerInfoOrNil else {
+                let alert = UIAlertController(title: "Address Not Found", message: "Please open Ethereum App on your Ledger device.", preferredStyle: .alert)
```

Comment:
looks like hardcoded strings to me

Reply:
yes, we use hardcoded strings everywhere